### PR TITLE
Ledger Api Test Tool: Add --command-submission-ttl-scale-factor arg.

### PR DESCRIPTION
**Pull request by gleber-da**
Scale factor for time-to-live of commands sent for ledger processing
(captured as Maximum Record Time in submitted transactions) for
"SemanticTests" suite. Useful to tune Maximum Record Time depending on
the environment and the Ledger implementation under test. Defaults to
1.0. Use numbers higher than 1.0 to make timeouts more lax, use
numbers lower than 1.0 to make timeouts more strict.

### Pull Request Checklist

- [x] Read and understand the [contribution guidelines](https://github.com/digital-asset/daml/blob/master/CONTRIBUTING.md)
- [x] Include appropriate tests
- [x] Set a descriptive title and thorough description
- [x] Add a reference to the [issue this PR will solve](https://github.com/digital-asset/daml/issues), if appropriate
- [x] Add a line to the [release notes](https://github.com/digital-asset/daml/blob/master/docs/source/support/release-notes.rst), if appropriate

NOTE: CI is not automatically run on non-members pull-requests for security
reasons. The reviewer will have to comment with `/AzurePipelines run` to
trigger the build.

### CODE REVIEW

File: ledger/ledger-api-integration-tests/src/test/lib/scala/com/digitalasset/platform/semantictest/SemanticTestAdapter.scala [hunk 1]
```diff
@@ -43,7 +44,7 @@ class SemanticTestAdapter(
 
   private def ledgerId = lc.ledgerId
 
-  private def ttlSeconds = 10
+  private def mrtSeconds = 10 * commandSubmissionTtlScaleFactor
```
Comment (meiersi-da) [inline ledger/ledger-api-integration-tests/src/test/lib/scala/com/digitalasset/platform/semantictest/SemanticTestAdapter.scala]: Why mrt and not ttl?

Comment (gleber-da) [reply ledger/ledger-api-integration-tests/src/test/lib/scala/com/digitalasset/platform/semantictest/SemanticTestAdapter.scala]: Done

File: docs/source/support/release-notes.rst [hunk 2]
```diff
@@ -12,7 +12,12 @@ HEAD — ongoing
 DAML Integration Kit
 ~~~~~~~~~~~~~~~~~~~~
 
-- Added new CLI flag ``--all-tests`` to the :doc:`Ledger API Test Tool </tools/ledger-api-test-tool/index>` to run all default and optional tests.
+- Added new CLI flag ``--all-tests`` to the :doc:`Ledger API Test Tool
+  </tools/ledger-api-test-tool/index>` to run all default and optional tests.
+- Added new CLI flag ``--command-submission-ttl-scale-factor`` to the
+  :doc:`Ledger API Test Tool </tools/ledger-api-test-tool/index>` to control scale
+  buffer time given for command submission via Max Record Time of submitted
+  transactions.
```
Comment (meiersi-da) [inline docs/source/support/release-notes.rst]: Copy docs from command line here. That explanation given there is better.

Comment (gleber-da) [reply docs/source/support/release-notes.rst]: Done

**Pull request merged**